Quintana_Jerrod/Assignments/flask_fundamentals/registration_form/templates/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Registraiton form project</title>
+    <link rel="stylesheet" href="{{url_for('static',filename='index.css')}}" media="screen" title="no title">
+  </head>
+  <body>
+    <div class="wrapper">
+
+      <form class="" action="/user" method="post">
+        First Name:<input type="text" name="first" value="">
+        Last Name:<input type="text" name="last" value="">
+        Email:<input type="text" name="email" value="">
+        Password: <input type="text" name="pass" value="">
+        Confirm Password: <input type="text" name="confirm" value="">
+        <input type="submit" name="submit" value="Enter">
+      </form>
+      {% with messages = get_flashed_messages() %}
```

Comment:
I prefer to have error messages show up on the TOP of the html page rather than the bottom; since when the user is sent back here, they will be looking at the top of the page. They shouldn't have to scroll down to see their error messages printed out.